tests/test_generate_file.py

```diff
@@ -82,6 +82,23 @@ def test_generate_file_jsonify_filter(env):
         assert json.loads(generated_text) == data
 
 
+@pytest.mark.usefixtures('remove_cheese_file')
+@pytest.mark.parametrize("length", (10, 40))
```

Comment:
you make matrix with 'length', but did not do with 'punctuation'

Reply:
Done 👍